Automattic/wp-calypso
# Bump `@wordpress/interface` and other monorepo packages for Gutenberg 13.3.0 compat

#### Changes proposed in this Pull Request

* Bump `@wordpress/interface` and other monorepo packages to the latest versions

Original discussion: p1653583842699789-slack-CBTN58FTJ

This one's tricky because `@wordpress/interface` isn't separately packaged in a way that lets us override a Core version. Instead, it's inlined by webpack rather than transformed into consuming a WP script by the dependency extraction plugin. Phew.

WordPress/gutenberg/pull/40923 introduced the new stuff, but Help Center is still packaging an older, outdated version, which then tramples the `core/interface` redux store namespace and lacks the new functions, which are called in the `edit-site` packages.

#### Testing instructions

On a sandboxed site with the `gutenberg-edge` sticker, build and run this version of ETK, and try to load the Site Editor.

Before you would see an error on a call to `setDefaultComplementaryArea` and a WSOD in the Site Editor.

Fixes #63944
